horbatiuk-vitalii/task_03/script_Linux_OS.sh

```
@@ -0,0 +1,31 @@
+    echo "$(date): Oversized folders found: $EXCLUDED_FOLDERS" | tee -a "$LOG_FILE"
+else
+    echo "$(date): No oversized folders found." | tee -a "$LOG_FILE"
+fi
+EXCLUDE_OPTS="--exclude='*.log' --exclude='.*' --exclude='/target' --exclude='/logs'"
+
+for folder in $EXCLUDED_FOLDERS; do
+    EXCLUDE_OPTS+=" --exclude='$folder'"
+done
+RSYNC_COMMAND="rsync -av --delete $EXCLUDE_OPTS '$SOURCE' '$DESTINATION'"
+
+echo "$(date): command executing: $RSYNC_COMMAND" | tee -a "$LOG_FILE"
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)